### PR TITLE
@broskoski: Signup Modal QA

### PR DESCRIPTION
**Pull request by craigspaeth**
This does a couple things:

* Get client-side + server-side tracking in sync
* Add a "full name" field above email
* Refresh the page after the thank you message on desktop to show logged in

Most importantly we needed to get client-side and server-side tracking in sync and both sending along the `user_id` and `acquisition_initiative`. The other two are should-have UX items.

cc @anipetrov

### CODE REVIEW

File: analytics/account_creation.js [hunk 1]
```diff
@@ -2,10 +2,48 @@
...
+
+    // 2. Store some data in cookies before being redirected everywhere
+    Cookies.set('analytics-signup', JSON.stringify({
+      service: 'facebook',
+      acquisition_initiative: location.search.replace('?m-id=', '')
+    }))
+  }
+)
+
+// 3. After landing back on Artsy send the tracking call and expire the cookie
```
Comment (broskoski) [inline analytics/account_creation.js]: Nice, thanks for the comments.

File: analytics/account_creation.js [hunk 2]
```diff
@@ -2,10 +2,48 @@
...
+if (Cookies.get('analytics-signup')) {
+  var data = JSON.parse(Cookies.get('analytics-signup'))
```
Comment (craigspaeth) [inline analytics/account_creation.js]: This explains the hoops we're jumping through to get our sophisticated signup spying working. Just like we need to persist some state in the session to follow the user around the web [on the server-side](https://github.com/artsy/artsy-passport/pull/88#discussion_r90137532), we do roughly the same thing here on the client-side using Cookies.js to remember the Facebook button click, then when the come back to Artsy after successful OAuth signup we make the tracking call here.

**Comment by broskoski** [comment]
Looks like a real ❌  but not sure what's going on there. Looks good otherwise!

File: components/marketing_signup_modal/index.jade [hunk 3]
```diff
@@ -18,7 +22,7 @@
     .marketing-signup-modal-error
   .marketing-signup-modal-divider Or
   a.avant-garde-button-black.marketing-signup-modal-fb(
-    href=sd.AP.facebookPath
+    href=sd.AP.facebookPath + '?acquisition_initiative=' + sd.MARKETING_SIGNUP_MODAL_SLUG
```
Comment (craigspaeth) [inline components/marketing_signup_modal/index.jade]: This is related to [this PR](https://github.com/artsy/artsy-passport/pull/88#discussion_r90137532) where we pass the acquisition initiative through to the server-side.

**Comment by anipetrov** [comment]
Is the expected behavior here to get both a client and a server event called `created_account` in the force_production segment project at each instance of account creation? I will QA and will watch at the data closely once this is deployed. 

Having complete double coverage will definitely make it possible for us to distinguish between server and client events in `force_production.created_account` but eventually it would be nice to have separate force and microgravity [server sources](https://segment.com/artsy-engineering/sources/catalog) set up in segment where we would send all our server side events. 

cc @wrgoldstein @joelauerbach

**Comment by craigspaeth** [comment]
Yep that's correct Ani. I haven't split the server-side events to a different account in this PR

**Pull request merged**